libs/utils/test.py

```diff
@@ -94,6 +94,10 @@ def _runExperiments(cls):
         experiments_conf = cls._getExperimentsConf(test_env)
         cls.executor = Executor(test_env, experiments_conf)
 
+        # Alias tests and workloads configurations
+        cls.confs = cls.executor.te.test_conf["confs"]
```

Comment:
The `confs` field is supposed to be part of the `experiments_conf` not the test_conf.